components/molecules/Menu.js

```diff
@@ -20,9 +20,14 @@ export function Menu({ loginText, items }) {
     >
       <div className="flex w-full justify-between py-6  md:py-1 mb-14 md:mb-0 lg:px-0">
         <div className="h-10 w-full sticky my-auto">
-          <h1 className="text-2xl pt-1 font-bold font-display ">
-            Service Canada
-          </h1>
+          <Link href="/home">
```

Comment:
This can still be a level 1 header. Will likely cause issues downstream since all of our header levels are based off of this one being h1

Can do something like `<Link><h1>link header</h1></Link>`